org.expresslrs.ExpressLRSConfigurator.yml

```diff
@@ -0,0 +1,72 @@
+app-id: org.expresslrs.ExpressLRSConfigurator
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "22.08"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node16
+command: expresslrs-configurator
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
+  - --socket=system-bus
```

Comment:
this is not allowed. You need to figure out which dbus names you need.

Reply:
Log only says
```
[3:1120/195125.745307:ERROR:bus.cc(398)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[3:1120/195125.745360:ERROR:bus.cc(398)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
```
so I might have to do some digging, but the app seems to startup fine.